### PR TITLE
zed: yoga merge

### PR DESCRIPTION
**Pull request by markgoddard**
- Bump Neutron tag to fix multiple bugs
- Bump Ceph images to latest stable releases
- Check if repos backup exists in kolla footer
- Still check for stackhpc repos in footer
- Apply suggestions from code review
- Fix bash in footer
- Fix libvirtd memory leak in CentOS Stream 8
- New neutron tag
- Fix cloudkitty_tag

### CODE REVIEW

File: releasenotes/notes/bump-ceph-images-f9d4721aa1d37729.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+upgrade:
+  - |
+    Updates default Ceph images to v17.2.7 for Quincy and v16.2.14 for Pacific.
```
Comment (markgoddard) [inline releasenotes/notes/bump-ceph-images-f9d4721aa1d37729.yaml]: Remove Pacific

**Comment by MoteHue** [comment]
Neutron building here: https://github.com/stackhpc/stackhpc-kayobe-config/actions/runs/6875515306

**Comment by MoteHue** [comment]
Neutron promotions: https://github.com/stackhpc/stackhpc-release-train/actions/runs/6876128040 https://github.com/stackhpc/stackhpc-release-train/actions/runs/6876123488

**Comment by markgoddard** [comment]
Trigger checks

**Review by MoteHue — APPROVED** [review]

**Pull request merged**